minesweeper.py

```diff
@@ -266,33 +273,28 @@ def flagging(event):
     "The force is strong with this one.",
 ]
 
-
-def highscore(mins, secs):
-    converted_to_seconds = mins * 60 + secs
-    for time in top_10_times:
-        if converted_to_seconds < time and len(top_10_times) < 10:
-            top_10_times.insert(
-                index, converted_to_seconds
-            )  # HOW DO I GET THE INDEX OF "TIME" IN FOR LOOP?
-        elif converted_to_seconds < time and len(top_10_times >= 10):
-            top_10_times.remove(top_10_times[-1])
-            top_10_times.insert(index, converted_to_seconds)  # SAME AS ABOVE
-
-
+def highscore():
+    data = where_this_file_is / 'high_scores.json'
+    if data.exists():
```

Comment:
Instead of this, it would be better to try to load the file, and if you get `FileNotFoundError`, just use the `json_dict` defined in the beginning of the file

Reply:
Added file handling logic